## E0611 (no-name-in-module)

### :x: Problematic code:

```console
cat > package/__init__.py <<EOF
class AudioTime:
    DECIMAL = 3
EOF

cat > package/AudioTime.py <<EOF
class AudioTime:
    pass


cat > test.py <<EOF
from package import AudioTime
print(AudioTime.DECIMAL)
EOF
```

### :heavy_check_mark: Correct code:

```python
cat > package/__init__.py <<EOF
class AudioTime:
    DECIMAL = 3
EOF

cat > test.py <<EOF
from package import AudioTime
print(AudioTime.DECIMAL)
EOF
```

### Rationale:

Used when a name cannot be found in a module.

### Related resources:

- [Testcases](https://github.com/PyCQA/pylint/blob/master/tests/regrtest_data/precedence_test.py)
- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22no-name-in-module%22+OR+%22E0611%22)
